SpotifyArtistsViewer/SASearchViewController.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  SASearchViewController.swift
+//  SpotifyArtistsViewer
+//
+//  Created by Kevin Zeckser on 6/6/16.
+//  Copyright © 2016 Kevin Zeckser. All rights reserved.
+//
+
+import UIKit
+
+class SASearchViewController: UITableViewController, UITextFieldDelegate {
+
+    // MARK: - Properties & Outlets
+    
+    @IBOutlet weak var textField: UITextField!
+    @IBOutlet weak var searchSegmentControl: UISegmentedControl!
+    private var artists = [SAArtist]()
+    private let requestManager = SARequestManager.sharedInstance
+ 
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.navigationController?.navigationBar.barTintColor = UIColor.blackColor()
+        self.view.backgroundColor = UIColor.blackColor()
+        textField.textColor = UIColor.whiteColor()
+    }
+    
+    // MARK: - SARequestManager Callback
+
+    private func updateFilteredArtistsWithResult(result: SAResponse) {
+        
+        switch result {
+        case .Failure(let error):
+            print("Error fetching artists: \(error)")
+        case .Success(let returnedArtists):
+            print("Success!!!")
+            self.artists = returnedArtists
+            self.tableView.reloadData()
+        }
+    }
+    
+    // MARK: - UITableViewDataSource
+
+    override internal func numberOfSectionsInTableView(tableView: UITableView) -> Int {
+        
+        return 1
+    }
+
+    override internal func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        
+        return self.artists.count
+        
+    }
+
+    override internal func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCellWithIdentifier("cell", forIndexPath: indexPath) as UITableViewCell?
+
+        cell?.textLabel?.text = self.artists[indexPath.row].name
+        return cell!
+    }
+    
+    // MARK: - UITextFieldDelegate
+    
+    @objc internal func textFieldShouldReturn(textField: UITextField) -> Bool {
+        
+        guard let searchQuery = self.textField.text else { print("yall's text field dun broke"); return true }
```

Comment:
put this else block on a new line(s).  You should never have to use a semicolon in swift.